### PR TITLE
enhancement: replace wallet address checkbox with disclaimer

### LINKED ISSUE #188: [Enhancement] checkbox titles and disclaimer
I think it makes more sense to transform the first checkbox in a disclaimer below the submit button which states: "When you click submit, your wallet address and public key will be transmitted to a smart contract stored permanently on the Gaia-X testnet."
Regarding the second check-box, could we please change the heading to "Metadata confirmation"?

### PR DESCRIPTION
**Pull request by LucaMilanese90**
Fixes #188

## Proposed Changes

  - transform "wallet address" checkbox to disclaimer
  - update "Metadata confirmation" checkbox title

### CODE REVIEW

File: src/components/pages/Publish/FormActions.tsx [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
         </Button>
-      )}
+
+        {status !== 'empty' && (
+          <Button style="text" size="small" onClick={resetFormAndClearStorage}>
+            Reset Form
+          </Button>
+        )}
+      </div>
+      <Alert state="info" text={walletDisclaimer} />
```
Comment (moritzkirstein) [inline src/components/pages/Publish/FormActions.tsx:43]: suggestion(non-blocking): I like that we use the `info` state here. However checking the preview it seems that warning & info use the same color scheme. Would expect something like our blue or gray for information, and use the orange for warning only.

I guess not high prio, only fix if this is small scope

**Review by moritzkirstein — APPROVED** [review]

File: src/components/pages/Publish/FormActions.tsx (hunk 1 above)
Comment (LucaMilanese90) [reply src/components/pages/Publish/FormActions.tsx:43]: I agree, I'll check if it's used somewhere else and update it with the gray color scheme, should take just a few minutes.

Comment (LucaMilanese90) [reply src/components/pages/Publish/FormActions.tsx:43]: Update: I checked and the `info' state is used in a few other places. The idea would be to turn those places to `warning` (to keep the current color) and just keep info for the new disclaimer under the buy button, would it be ok for you?

Comment (moritzkirstein) [reply src/components/pages/Publish/FormActions.tsx:43]: Yes, sounds right! Let's update both new disclaimers to info scheme (publishing and consuming)

**Review by moritzkirstein — APPROVED** [review]

**Pull request merged**